tests/testthat/test-root.R

```diff
@@ -178,6 +178,23 @@ test_that("is_git_root", {
                  "No root directory found.* a directory `.*`"),
     TRUE
   )
+
+  # Copy .git dir to a separate location, then make a .git file.
+  # (other_git_folder becomes a bare git repo)
+  old_git_location <- file.path(wd, "git", ".git")
+  new_git_location <- file.path(wd, "other_git_folder")
+  file.rename(old_git_location, new_git_location)
+  writeLines(paste("gitdir:", new_git_location), old_git_location)
+  with_mock(
```

Comment:
I'd greatly appreciate if you could switch to mockr, also for the other tests. The original `with_mock()` is likely to break sooner than later.